### ISSUE TITLE
Choosing files when adding a new connection

### ISSUE THREAD

**Issue by simran-sohanpal** [IBM-Blockchain/blockchain-vscode-extension#326]
When adding a connection profile, certificate and key would it be possible to identify only the files that are in the correct format? e.g. the connection profile needs to be in .json format so only files in .json format are available to select 

Could be just a small thing but thought it would help with efficiency in choosing files

**Comment by heatherlp** [comment]
Yep we can do this, we just need to change browseEdit() in UserInputUtil to make use of the filters parameter of showOpenDialog
